pkg/verifier/util/utils.go

```diff
@@ -2,42 +2,49 @@ package util
 
 import (
 	"context"
+	"net/url"
 
 	"github.com/bacalhau-project/bacalhau/pkg/model"
+	"github.com/bacalhau-project/bacalhau/pkg/publisher"
 	"github.com/bacalhau-project/bacalhau/pkg/system"
 	"github.com/bacalhau-project/bacalhau/pkg/verifier"
 	"github.com/bacalhau-project/bacalhau/pkg/verifier/deterministic"
+	"github.com/bacalhau-project/bacalhau/pkg/verifier/external"
 	"github.com/bacalhau-project/bacalhau/pkg/verifier/noop"
+	"go.uber.org/multierr"
 )
 
 func NewStandardVerifiers(
```

Comment:
I think we can simplify the error handling of this method: return an error if it occurs rather than append them all together.

Reply:
See prev comment about preferring multiple errors from independent operations.